<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabWidget</class>
 <widget class="QTabWidget" name="tabWidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>937</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="Spectrum">
   <attribute name="title">
    <string>Tab 1</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QTabWidget" name="horizontalTabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="horizontalTabWidgetPage1">
       <attribute name="title">
        <string>400МГц</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Новый столбец</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>F_start</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>data</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>800МГц</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>301</width>
          <height>769</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F_start</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>data</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>1200МГц</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>301</width>
          <height>769</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F_start</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>data</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>2400МГц</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>291</width>
          <height>769</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F_start</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>data</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Частоти</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QGroupBox" name="verticalGroupBox">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="2" column="1">
        <widget class="PlotWidget" name="graphicsView2400">
         <property name="cursor" stdset="0">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="L_Widget" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777214</width>
           <height>88</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_L">
            <item>
             <property name="text">
              <string>Широкосмугова</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Полігармоніка</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Повторювальна</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_L">
            <property name="text">
             <string>Фільтр</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_L"/>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_L">
            <property name="text">
             <string>Усереднення</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Демодуляція</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Очист. MaxHold</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="PlotWidget" name="graphicsView800">
         <property name="cursor" stdset="0">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PlotWidget" name="graphicsView400">
         <property name="cursor" stdset="0">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="UHF_Wiget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_UHF">
            <property name="text">
             <string>Фільтр</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_UHF">
            <property name="text">
             <string>Усереднення</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_UHF">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>90</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Відсутня</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Широкосмугова</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_UHF"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="demod_400">
            <property name="text">
             <string>Демодуляцiя</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="reset_maxhold">
            <property name="text">
             <string>Очист. MaxHold</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QWidget" name="S_Widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>16</number>
          </property>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_S">
            <item>
             <property name="text">
              <string>Широкосмугова</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Прицільна</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Повторювальна</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_S">
            <property name="text">
             <string>Фільтр</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_S">
            <property name="text">
             <string>Усереднення</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_S"/>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>Демодуляція</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>Очист. Maxhold</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="UHF_L_Widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_UHF_L">
            <property name="text">
             <string>Фільтр</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_UHF_L">
            <item>
             <property name="text">
              <string>Широкосмугова</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_UHF_L">
            <property name="text">
             <string>Усереднення</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_UHF_L"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Демодуляція</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Очист.Maxhold</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="PlotWidget" name="graphicsView1000">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>53</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="statusTip">
       <string notr="true"/>
      </property>
      <property name="whatsThis">
       <string notr="true"/>
      </property>
      <property name="accessibleName">
       <string notr="true"/>
      </property>
      <property name="accessibleDescription">
       <string notr="true"/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="reciev">
         <property name="text">
          <string>Прийом</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="transmitt">
         <property name="text">
          <string>Завада</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>verticalGroupBox</zorder>
   <zorder>verticalGroupBox</zorder>
   <zorder>horizontalTabWidget</zorder>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>Страница</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="PlotWidget" name="graphicsView"/>
    </item>
    <item>
     <widget class="PlotWidget" name="graphicsView_2">
      <property name="minimumSize">
       <size>
        <width>1012</width>
        <height>440</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_3">
   <attribute name="title">
    <string>Страница</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Tab 2</string>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
